Makefile

```diff
@@ -17,28 +17,53 @@
 #
 # Makefile with some common workflow for dev, build and test
 #
+export GOPROXY?=https://proxy.golang.org/
+CONTROLLER_GEN_BIN_PATH := $(shell which controller-gen)
+.DEFAULT_GOAL:=help
```

Comment:
I'd prefer to prune this (the default behavior for make is generally assumed to be build, and it's easier to work with that way)